doc/source/building/index.rst

```diff
@@ -0,0 +1,354 @@
+    LAPACK libraries, and pkg-config. They're easiest to install with
+    `Homebrew <https://brew.sh/>`__::
+
+        brew install gfortran openblas pkg-config
+
+    .. note::
+
+        As of SciPy >=1.2.0, we do not support compiling against the system
+        Accelerate library for BLAS and LAPACK. It does not support a sufficiently
+        recent LAPACK interface. This is planned to change in 2023, because macOS
+        13.3 introduced a major upgrade to Accelerate which resolved all known
+        issues.
+
+  .. tab-item:: Windows
+
+    A compatible set of C, C++ and Fortran compilers is needed to build SciPy.
+    This is tricker on Windows than on other platforms, because MSVC does not
+    support Fortran, and gfortran and MSVC can't be used together. You will
+    need one of these sets of compilers:
+
+    1. MSVC + Intel Fortran (``ifort``)
+    2. Intel compilers (``icc``, ``ifort``)
+    3. Mingw-w64 compilers (``gcc``, ``g++``, ``gfortran``)
+
+    Compared to macOS and Linux, building SciPy on Windows is more difficult,
+    largely due to the difficulty of setting up these compilers. It is not
+    possible to just call a one-liner on the command prompt as you would on
+    other platforms.
+
+    First, install Microsoft Visual Studio - the 2019 Community Edition or any
+    newer version will work (see the
+    `Visual Studio download site <https://visualstudio.microsoft.com/downloads/>`__). 
+    This is needed even if you use the MinGW-w64 or Intel compilers, in order
+    to ensure you have the Windows Universal C Runtime.
+
+    .. tab-set::
+
+      .. tab-item:: MSVC
+
+        The MSVC installer does not put the compilers on the system path, and
+        the install location may change. To query the install location, MSVC
+        comes with a ``vswhere.exe`` command-line utility. And to make the
+        C/C++ compilers available inside the shell you are using, you need to
+        run a ``.bat`` file for the correct bitness and architecture (e.g., for
+        64-bit Intel CPUs, use ``vcvars64.bat``).
+
+        For detailed guidance, see `Use the Microsoft C++ toolset from the command line
+        <https://learn.microsoft.com/en-us/cpp/build/building-on-the-command-line?view=msvc-170>`__.
+
+      .. tab-item:: Intel
+
+        Similar to MSVC, the Intel compilers are designed to be used with an
+        activation script (``Intel\oneAPI\setvars.bat``) that you run in the
+        shell you are using. This makes the compilers available on the path.
+        For detailed guidance, see
+        `Get Started with the Intel® oneAPI HPC Toolkit for Windows
+        <https://www.intel.com/content/www/us/en/docs/oneapi-hpc-toolkit/get-started-guide-windows/2023-1/overview.html>`__.
+
+      .. tab-item:: MinGW-w64
+
+        There are several sources of binaries for MinGW-w64. We recommend the
+        RTools versions, which can be installed with::
+
+            choco install rtools
```

Comment:
Maybe link to Chocolatey installation instructions?

Reply:
Also I just tried `choco install rtools` instead of `choco install rtools -y --no-progress --force --version=4.0.0.20220206` and I was not successful in building. Maybe pin the version?

Reply:
The latest version of rtools available via choco places the build tools in a different location to that which we use in 4.0.0.20220206. I've not looked into where they actually go with the new version. I think that the newer version uses gcc12, the older version gcc10

Reply:
Also, choco can be used to install pkg-config.

Reply:
Yes, there were two locations with `gfortran.exe`: `rtools43\x86_64-w64-mingw32.static.posix\bin` and `rtools43\usr\lib\mxe\usr\x86_64-w64-mingw32.static.posix\bin`. I chose the one with the shorter name.

Reply:
Is it `choco install pkgconfiglite`? I guess that is needed only when not using conda.

Reply:
I'll leave this open for @andyfaff, but it's resolved from my perspective. I don't know about pkg-config.

Reply:
It's only needed when not using conda indeed, I'll add a comment behind the command.

Reply:
In the windows CI pkg-config is already installed. I'm not sure if it's part of rtools. Pkgconfiglite may be the one.

Reply:
From the Rtools description in Chocolatey:

_Rtools is a toolchain bundle used for building R packages from source (those that need compilation of C/C++ or Fortran code) and for build R itself. Rtools43 is used for R-devel, the development version of R since revision 83535 and is intended for R 4.3.0._

_Rtools43 consists of Msys2 build tools, GCC 12/MinGW-w64 compiler toolchain, libraries built using the toolchain, and QPDF. Rtools43 supports 64-bit Windows and UCRT as the C runtime._

_Compared to Rtools42, Rtools43 has newer versions of three core components: GCC, MinGW-w64, and binutils. It is therefore recommended to re-compile all code with the new toolchain to avoid problems. The code compiled by even earlier versions of Rtools is incompatible due to use of MSVCRT and has to be recompiled with Rtools43 for use in R packages._

So that's not completely clearcut - `pkg-config` is not part of `binutils`. I downloaded the built sources as a zipfile from https://cran.r-project.org/bin/windows/Rtools/rtools43/files/, and in `installed.list` we find `pkgconf 1.8.0`. So yes, that's it - it is included. Hence I can remove this one line again.

Reply:
I believe that's the conclusive answer, so I'll resolve this.